src/client/datascience/notebook/notebookControllerManager.ts

```diff
@@ -257,9 +257,6 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
         document: NotebookDocument,
         token: CancellationToken
     ): Promise<KernelConnectionMetadata | undefined> {
```

Comment:
I was basically doubling up on this call. Instead just rely on the call in setPreferredController, findPreferredConnection doesn't need controllers to operate.